commcare_connect/commcarehq/api.py

```
@@ -84,6 +86,66 @@ def create_or_update_case_by_work_area(work_area: WorkArea) -> CommCareCase:
+
+    cases_data = []
+    for wa in work_areas:
+        case_data = dict(WorkAreaCaseSerializer(wa).data)
+        case_data["owner_id"] = owner_id_by_username[wa.opportunity_access.user.username.lower()]
+        case_data["create"] = None  # UPSERT: HQ decides create vs update via external_id
+        cases_data.append(case_data)
+
+    cases = bulk_create_or_update_cases(api_key, domain, cases_data)
+
+    newly_created = []
+    for wa, case in zip(work_areas, cases):
```

Ruff (0.15.12)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)